sync/src/relayer/tests/helper.rs

```diff
@@ -0,0 +1,144 @@
+use crate::{Relayer, SyncSharedState};
```

Comment:
YFI, I am planning to move `sycn/src/relayer/tests` into `sync/src/tests/`, after https://github.com/nervosnetwork/ckb/pull/1204.

Reply:
And can reuse the test utils.